src/projections/igh_o.cpp

```diff
@@ -0,0 +1,235 @@
+#define PJ_LIB__
+
+#include <errno.h>
+#include <math.h>
+
+#include "proj.h"
+#include "proj_internal.h"
+
+PROJ_HEAD(igh_o, "Interrupted Goode Homolosine Oceanic View") "\n\tPCyl, Sph";
+
+C_NAMESPACE PJ *pj_sinu(PJ *), *pj_moll(PJ *);
+
+/* 40d 44' 11.8" [degrees] */
+/*
+static const double d4044118 = (40 + 44/60. + 11.8/3600.) * DEG_TO_RAD;
+has been replaced by this define, to eliminate portability issue:
+Initializer element not computable at load time
+*/
+#define d4044118 ((40 + 44/60. + 11.8/3600.) * DEG_TO_RAD)
```

Comment:
Could this get a more descriptive name? I suspect this is a longitude with a special property but it is not entirely clear from the comment here. 

It would also be better to da a `static const double ...` as below

Reply:
Again, maintaining consistency with `igh.cpp`

Reply:
d4404118 is not a longitude - its the latitude of the breakpoint between the sinusoidal and Mollweide projections.

Reply:
Please document that in a comment, so the next guy has a chance to figure out what's going on

Reply:
Sure - I just committed some more documentation describing the latitude constant and the integer numbering scheme.

Reply:
> `static const double ...` as below

The C++11 canoncial way for such compile time constants is:
constexpr double = ....